helprequest/views.py

```diff
@@ -73,3 +86,20 @@ def form_valid(self, form):
         # form.send_email()
         return super().form_valid(form)
 
+
+@method_decorator(login_required, name='dispatch')
+class HelpRequestUpdate(UpdateView):
+    model = HelpRequest
+    fields = [
+        # "slug"
+        "medic_notes"
+    ]
+    template_name = 'help_response.html'
+    success_url = '/'
+        
+
+def form_valid(self, form):
```

Comment:
Where is this function being called/used? Also, is it inside the HelpRequestUpdate class or outside - it looks the indentation is wrong :) 
Also, if you're not adding any functionality to the function, there is no need to override it